app/src/views/CountryNsOverviewContextAndStructure/index.tsx

```diff
@@ -80,13 +83,27 @@ export function Component() {
                             href={countryResponse?.society_url}
                             external
                             withLinkIcon
+                            variant="primary"
                         >
                             {resolveToString(
                                 strings.countryRCHomepage,
                                 { countryName: countryResponse?.name ?? '-' },
                             )}
                         </Link>
                     )}
+                    {isTruthyString(countryResponse.disaster_law_url) && (
+                        <Link
+                            href={countryResponse?.disaster_law_url}
```

Comment:
`countryResponse.disaster_law_url` is already accessed above without the optional chaining, so countryResposne should already be defined
```suggestion
                            href={countryResponse.disaster_law_url}
```